settings.py

```diff
@@ -18,3 +18,5 @@
 USE_TZ = False
 
 INSTALLED_APPS = ("db",)
+
+ALLOWED_HOSTS = ["*"]
```

Comment:
Setting `ALLOWED_HOSTS` to `['*']` is acceptable for development but poses a security risk in production. Consider specifying allowed hosts when deploying to a production environment.